src/firefly/python/display/FireflyClient.py

```diff
@@ -533,11 +621,41 @@ def createRangeValuesZScale(algorithm, zscaleContrast=25, zscaleSamples=600, zsc
         :param zscaleSamplesPerLine: zscale sample per line
         :return: a serialized range values string
         """
-        retval= FireflyClient.createRV('zscale',1,2,algorithm,zscaleContrast, zscaleSamples, zscaleSamplesPerLine)
+        retval= FireflyClient.createRV('zscale',1,1,algorithm,zscaleContrast, zscaleSamples, zscaleSamplesPerLine)
         if not retval:
             a= algorithm if algorithm.lower() in FireflyClient.stretchAlgorithmDict else 'linear'
             retval= FireflyClient.createRV('zscale',1,2,a,25,600,120)
         return retval
 
 
+    @staticmethod
+    def genItemId(item):
+        """
+        generate an ID for some entity like 'Table', 'RegionLayer', 'Extension'
+        :param item: entity type
+        :return: an ID string
+        """
 
+        if item in FireflyClient.itemID:
+            FireflyClient.itemID[item] += 1
+            return item + '-' + str(FireflyClient.itemID[item])
+        else:
+            return None
+
+    @staticmethod
+    def createImageURL(imageSource):
```

Comment:
I'm wondering if this should be called (or made into) `downloadPng` because originally I thought it meant "give me a link to the FITS file".